### PR TITLE
Implement UserAgent parser

### PR DESCRIPTION
**Pull request by gayashanbc**
Related https://github.com/wso2/product-is/issues/6012

### CODE REVIEW

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/pom.xml [hunk 1]
```diff
@@ -110,6 +118,8 @@
                                 <bundleDef>org.wso2.carbon.identity.framework:org.wso2.carbon.identity.base</bundleDef>
                                 <bundleDef>org.wso2.carbon.identity.framework:org.wso2.carbon.identity.core</bundleDef>
                                 <bundleDef>org.wso2.carbon.identity.framework:org.wso2.carbon.identity.application.common</bundleDef>
+                                <bundleDef>ua.parser.wso2:ua-parser:${ua_parser.version}</bundleDef>
+                                <bundleDef>commons-collections.wso2:commons-collections:${apache.common.collection.version}</bundleDef>
```
Comment (senthalan) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/pom.xml]: No need to pack commons-collections.wso2:commons-collections, as it is already packed in the product. Pls check

File: components/identity-core/org.wso2.carbon.identity.core/pom.xml [hunk 2]
```diff
@@ -177,6 +181,7 @@
 
                             org.xml.sax,
                             org.apache.commons.collections; version="${commons-collections.wso2.osgi.version.range}",
+                            ua_parser.*,
```
Comment (senthalan) [inline components/identity-core/org.wso2.carbon.identity.core/pom.xml]: No need to import all the sub-packages.

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/pom.xml (hunk 1 above)
Comment (gayashanbc) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/pom.xml]: Resolved

File: components/identity-core/org.wso2.carbon.identity.core/pom.xml (hunk 2 above)
Comment (gayashanbc) [reply components/identity-core/org.wso2.carbon.identity.core/pom.xml]: Resolved

**Pull request merged**